lib/discovery/index.js

```diff
@@ -9,7 +9,7 @@ import base32 from 'base32.js'
 /**
  * @typedef {Object} DiscoveryEvents
  * @property {(connection: NoiseSecretStream<RawConnectionStream>, info: PeerInfo) => void} connection
- * @property {(topicStatus: { topic: TopicString, mdns: TopicServiceStatus, dht: TopicServiceStatus }) => void} topicStatus
+ * @property {(topicStatus: { topic: TopicId, mdns: TopicServiceStatus, dht: TopicServiceStatus }) => void} topicStatus
```

Comment:
Did a handful of renaming things in here to match the pattern that I ended up with in authstore:

- `<entityname>Key` or `<entityname>PublicKey` (if part of a keypair) for buffers
- `<entityname>Id` for hex string encoding of public key buffer